<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetPattern</class>
 <widget class="QDialog" name="GetPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QProgressBar" name="PrgBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnApplay">
       <property name="text">
        <string>Applay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnUndo">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbCurrentPath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
